app/src/main/java/com/example/demo/kotlin_practice/interfacePractice/IntDemo.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.demo.kotlin_practice.interfacePractice
+
+class IntDemo : A, B {
```

Comment:
add line

Reply:
`IntDemo`!!! Do not be lazy while giving names to entities or variables, always take time and give proper name